ironfish/src/network/peerNetwork.ts

```diff
@@ -309,7 +309,13 @@ export class PeerNetwork {
           return
         }
 
-        this.peerManager.createPeerFromInboundWebSocketConnection(connection, address)
+        let wsAddress: WebSocketAddress | null = null
```

Comment:
What do you think about creating a function for going from address/port -> WebSocketAddress in the util file?

Reply:
yea I think we could def change the parseUrl a bit to output that. Because that function should probably never return `null` for the `hostname`. If there's no `hostname` it's prob invalid. I'll try to do that in a separate PR